2021-04-22 07:41:54,479 INFO    MainThread:18994 [wandb_setup.py:_flush():69] setting env: {}
2021-04-22 07:41:54,479 INFO    MainThread:18994 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-22 07:41:54,479 INFO    MainThread:18994 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210422_074154-33uinaa3/logs/debug.log
2021-04-22 07:41:54,479 INFO    MainThread:18994 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210422_074154-33uinaa3/logs/debug-internal.log
2021-04-22 07:41:54,479 INFO    MainThread:18994 [wandb_init.py:init():369] calling init triggers
2021-04-22 07:41:54,479 INFO    MainThread:18994 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-22 07:41:54,479 INFO    MainThread:18994 [wandb_init.py:init():418] starting backend
2021-04-22 07:41:54,479 INFO    MainThread:18994 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-22 07:41:54,493 INFO    MainThread:18994 [backend.py:ensure_launched():123] starting backend process...
2021-04-22 07:41:54,498 INFO    MainThread:18994 [backend.py:ensure_launched():128] started backend process with pid: 19019
2021-04-22 07:41:54,500 INFO    MainThread:18994 [wandb_init.py:init():423] backend started and connected
2021-04-22 07:41:54,502 INFO    MainThread:18994 [wandb_init.py:init():465] updated telemetry
2021-04-22 07:41:54,502 INFO    MainThread:18994 [wandb_init.py:init():484] communicating current version
2021-04-22 07:41:55,363 INFO    MainThread:18994 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-22 07:41:55,364 INFO    MainThread:18994 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-22 07:41:55,571 INFO    MainThread:18994 [wandb_init.py:init():522] starting run threads in backend
2021-04-22 07:41:55,601 INFO    MainThread:18994 [wandb_run.py:_console_start():1524] atexit reg
2021-04-22 07:41:55,601 INFO    MainThread:18994 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-22 07:41:55,602 INFO    MainThread:18994 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-22 07:41:55,605 INFO    MainThread:18994 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-22 07:41:55,605 INFO    MainThread:18994 [wandb_init.py:init():546] run started, returning control to user process
2021-04-22 07:42:12,139 INFO    MainThread:18994 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-22 07:42:12,140 INFO    MainThread:18994 [wandb_run.py:_restore():1466] restore
2021-04-22 07:42:14,348 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 574
  total_bytes: 574
}

2021-04-22 07:42:14,587 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 574
  total_bytes: 4741
}

2021-04-22 07:42:14,978 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4741
  total_bytes: 4741
}

2021-04-22 07:42:15,081 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4741
  total_bytes: 4741
}

2021-04-22 07:42:15,183 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4741
  total_bytes: 4741
}

2021-04-22 07:42:15,286 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4741
  total_bytes: 4741
}

2021-04-22 07:42:15,388 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4741
  total_bytes: 4741
}

2021-04-22 07:42:15,490 INFO    MainThread:18994 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4741
  total_bytes: 4741
}

2021-04-22 07:42:16,589 INFO    MainThread:18994 [wandb_run.py:_show_files():1838] logging synced files
